hedera-node/src/main/java/com/hedera/services/context/NodeInfo.java

```diff
@@ -0,0 +1,55 @@
+package com.hedera.services.context;
+
+import com.swirlds.common.AddressBook;
+
+import java.util.function.Supplier;
+
+/**
+ * Summarizes useful information about the nodes in the {@link AddressBook}
+ * from the Platform. In the future, there may be events that require
+ * re-reading the book; but at present nodes may treat the initializing
+ * book as static.
+ */
+public class NodeInfo {
+	private boolean bookIsRead = false;
+
+	private int n;
```

Comment:
nit: I know `n` is an easy choice, but I think `numberOfNodes` will increase readability.